src/main/scala/faces/utils/Helpers.scala

```diff
@@ -64,7 +69,45 @@ case class Helpers(cfg: FacesSettings)(implicit rnd: Random) {
   }
 
   // the renderer for the model
-  val renderer: MoMoRenderer = MoMoRenderer(model, RGBA.BlackTransparent).cached(5)
+  val renderer: CorrespondenceMoMoRenderer = CorrespondenceMoMoRenderer(model, RGBA.BlackTransparent).cached(5)
+
+  val renderingMethods = {
+    val depthMapGenerator = new DepthMapRenderer(renderer)
+    val colorCorrespondenceGen = new CorrespondenceColorImageRenderer(renderer)
+    //additional rendering methods
+    val dm = if(cfg.renderingMethods.renderDepthMap) {
+      Some(("_depth", depthMapGenerator))
+    }else None
+    val cm = if(cfg.renderingMethods.renderColorCorrespondenceImage) {
+      Some(("_correspondence", colorCorrespondenceGen))
+    }else None
+    val rn = if(cfg.renderingMethods.render) {
+      Some(("", renderer))
+    }else None
+    IndexedSeq(rn, dm, cm).flatten
+  }
+
+  val aflwLmTags = Seq(
```

Comment:
It would be much more useful if people could pass in the landmarks they want. We have landmark formats for this 
 which you can use to specify landmarks on the reference. If those are just the default ones, then I can live with them being here hardcoded. But I would also prefer to show to the users that they can be changed and hence provide a file.